### PR TITLE
misc(treemap): duplicate-modules view mode

### PR DESCRIPTION
**Pull request by connorjclark**
https://lighthouse-git-treemap-duplicate-googlechrome.vercel.app/gh-pages/treemap/?debug

image

ref #11254

### CODE REVIEW

File: lighthouse-treemap/app/src/main.js [hunk 1]
```diff
@@ -208,6 +211,63 @@ class TreemapViewer {
...
+
+      /** @type {LH.Treemap.NodePath[]} */
+      const highlightNodePaths = [];
+      for (const nodesWithSameModuleName of moduleNameToNodes.values()) {
+        if (nodesWithSameModuleName.length === 1) continue;
+
+        const bytes = [];
+        for (const {node} of nodesWithSameModuleName) {
+          bytes.push(node.resourceBytes);
+          console.log(node.duplicatedNormalizedModuleName);
```
Comment (adamraine) [inline lighthouse-treemap/app/src/main.js:242]: Is this log necessary?

File: lighthouse-treemap/app/src/main.js [hunk 2]
```diff
@@ -208,6 +211,63 @@ class TreemapViewer {
...
+        for (let i = 1; i < bytes.length; i++) duplicatedBytes += bytes[i];
+        if (duplicatedBytes < DUPLICATED_MODULES_IGNORE_THRESHOLD) continue;
+
+        for (const {path} of nodesWithSameModuleName) {
+          highlightNodePaths.push([root.name, ...path]);
+        }
+        potentialByteSavings += duplicatedBytes;
+      }
+
+      // TODO: Show button but disable?
```
Comment (adamraine) [inline lighthouse-treemap/app/src/main.js:257]: I would prefer this to not showing it personally.

Comment (connorjclark) [reply lighthouse-treemap/app/src/main.js:257]: yeah I think so too

**Review by adamraine — APPROVED** [review]

**Comment by mspaansen** [comment]
Ok

**Review by mspaansen — CHANGES_REQUESTED** [review]
Ok

File: lighthouse-treemap/app/src/main.js (hunk 2 above)
Comment (mspaansen) [reply lighthouse-treemap/app/src/main.js]: I hope

**Pull request merged**